api_practice/main.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import FastAPI
```

Comment:
As requested in the first US, we need the package to be called "app" instead of" "api_practice". Please rename this also.